### ISSUE TITLE
Try name and title attributes when rendering resource h1

### ISSUE THREAD

**Issue by guntis** [cubesystems/releaf#295]
Current implementation from `Releaf::Builders::Base`:

```
def resource_to_text(resource)
  resource.send(resource.respond_to?(:to_text) ? :to_text : :to_s)
end
```

This method is used in h1 headings in admin. 

Currently, to display the title of the object, it is needed to define `#to_text` just for this purpose. This could be improved by automatically looking for typical title attributes and using them if they are present.

The method should look for possible methods in this order: `:to_text`, `:name`, `:title`, `:to_s`

**Comment by graudeejs** [comment]
Great idea

**Comment by miks** [comment]
Fixed with #303
